xonsh/completer.py

```diff
@@ -7,12 +7,12 @@
 import subprocess
 import sys
 
+from builtins import __xonsh_env__ as ENV
```

Comment:
You probably shouldn't do this at import because `__xonsh_env__` hasn't been added yet.  It better to set 

```python
env = builtins.__xonsh_env__
```

in the function scope where you need it.